Sources/StreamChatUI/ChatMessageList/MessageComposer/ChatMessageSendButton_Tests.swift

```diff
@@ -0,0 +1,85 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import StreamChatTestTools
+@testable import StreamChatUI
+import SwiftUI
+import XCTest
+
+class ChatMessageSendButton_Tests: XCTestCase {
+    private lazy var container = UIView().withoutAutoresizingMaskConstraints
+
```

Comment:
Why do we need this container ? 🤔  Isn't providing a size on the AssertSnapshot enough? Or set constraints of with and height of the button?